### ISSUE TITLE
(Document how to) use runtime only build of vue

### ISSUE THREAD

**Issue by SebastianS90** [vuejs/laravel-elixir-vue-2#6]
When using `require(vue)`, then the Vue package documents that the runtime only build `vue.common.js` is included, as the `.vue` files shall be precompiled by webpack.

This package changes that behavior because it [sets an alias to use `vue.js` instead](https://github.com/vuejs/laravel-elixir-vue-2/blob/5304017a58a4c496df4748862397742be621981d/index.js#L8)

Note that `vue.common.js` contains several switches `process.env.NODE_ENV !== 'production'` to disable debugging code for production. When running `gulp --production`, then webpack injects `"production"`  for these `process.env.NODE_ENV` variables. This ensures that the debugging branches are always false and minification (uglifyjs) will remove them.

But the `vue.js` file which will be included when using this package does not have these variables, the value is hardcoded to `"development"` thus enabling all debugging code regardless of the `--production` switch being present.

You can workaround that issue by explicitly requiring the `vue.common.js` file within the `node_modules` folder. You should at least document that this package sets the alias and how to workaround it, or in my opinion, remove the alias completely as the default behavior defined by the Vue package should fit most use cases.

**Comment by v-jacob** [comment]
I agree. I don't see why **vue** needs to be aliased to **vue/dist/vue** since templates will be compiled to render functions.

In the mean time, you can do this to override the alias by passing in webpack options as the 4th argument to mix.webpack

```
let webpackOpt = {
    resolve: {
        alias: {
            vue: 'vue'
        }
    }
};
```

```
elixir(mix => {
    mix.webpack('app.js', '', '', webpackOpt);
});
```

**Comment by SebastianS90** [comment]
Fixed a long time ago in #23
